src/components/IndexTable/IndexTable.scss

```diff
@@ -224,6 +241,39 @@ $loading-panel-height: rem(53px);
   }
 }
 
+.Table-unselectable {
+  .TableCell:first-child {
+    left: 0;
+    background-color: var(--p-surface);
+    position: sticky;
+    z-index: z-index(sticky-cell, $index-table-stacking-order);
+  }
+
+  .statusSuccess {
+    .TableCell:first-child {
+      background-color: var(--p-surface-primary-selected);
+    }
+  }
+
+  .statusSubdued {
+    .TableCell:first-child {
+      background-color: var(--p-surface-subdued);
+    }
+  }
+
+  .TableRow-hovered {
+    .TableCell:first-child {
+      background-color: var(--p-surface-selected-hovered);
+    }
+  }
+
+  .TableRow-selected {
```

Comment:
Is there ever a case where we could have both `Table-unselectable` and `TableRow-selected` or could we remove this 🤔